src/popup/components/TransferSendForm.vue

```diff
@@ -294,14 +293,14 @@ export default defineComponent({
     function getMessageByFieldName(fieldName: string): IInputMessage {
       if (!errors.value) return { status: 'success' };
 
-      const items = errors.value[fieldName as keyof typeof errors.value];
-      if (items) {
-        // eslint-disable-next-line no-console
-        console.log(items);
-        // TODO: Vee-validate v4 does not return error codes with errors
-        // We need to find a new way to filter messages
-        throw new Error('Function needs refactoring.');
-      }
+      // const items = errors.value[fieldName as keyof typeof errors.value];
```

Comment:
Why do we need to keep them as commented code here?

Reply:
As mentioned above in the description of this PR, I've commented out code that is filtering errors since that is no longer possible and it will be fixed in a separate PR

Reply:
Ok LGTM then.